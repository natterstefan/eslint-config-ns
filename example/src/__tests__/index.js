import React from 'react'
import { shallow } from 'enzyme'

import { App } from '../index'

describe('App', () => {
  test('renders', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
