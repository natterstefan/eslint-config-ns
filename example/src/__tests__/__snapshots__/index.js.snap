// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<Fragment>
  <h1>
    Some headline
  </h1>
  <Text />
</Fragment>
`;
